man/tpm2_startauthsession.1.md

```diff
@@ -0,0 +1,71 @@
+% tpm2_startauthsession(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_startauthsession**(1) - Start a session with the TPM.
+
+# SYNOPSIS
+
+**tpm2_startauthsession** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_startauthsession**(1) Starts a session with the TPM. The default is
+to start a *trial* session unless the **-a** option is specified. It outputs
+the session handle in a yaml format to stdout as key value "session-handle"
+and the handle in hex (0x1234) format.
+
+# OPTIONS
+
+  * **-a**, **--auth-policy-session**:
+
+    Change the policy session from a *trial* session to a *policy* session.
+    **NOTE**: A *trial* session is used when building a policy and a *policy*
+    session is used when authenticating with a policy.
```

Comment:
Nice note.

Reply:
Tools, like tpm2_unseal() could actually check for the session type in the session file and issue an error before the RC attribute error that the tpm will return. That is much more user friendly IMO. Something like, "the session being used for this is a trial session, where a policy session was expected."